experimental/dsn/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"path/filepath"
+	"runtime"
+
+	"github.com/jinfwhuang/ds-toolkit/experimental/dsn/arweave"
+	"github.com/jinfwhuang/ds-toolkit/experimental/dsn/ceramic"
+	estuary "github.com/jinfwhuang/ds-toolkit/experimental/dsn/filecoin-estuary"
+	"github.com/jinfwhuang/ds-toolkit/experimental/dsn/sia"
+	"github.com/jinfwhuang/ds-toolkit/experimental/dsn/storj"
+)
+
+var (
+	data = []byte("123")
+)
+
+func main() {
+}
+
+func arweaveExample() {
+	id, err := arweave.Write(data, fileInRuntimeDir("./arweave/wallet.json"))
+	if err != nil {
+		println(err.Error())
+	}
+	println(id)
+
+	// This is an example id that should be retrieved from Write function.
+	// Arweave takes couple of minutes until the id is accepted from the chain,
+	// that is why we use another random id that is already in the chain here.
+	ret, err := arweave.Read("IUYBL-mW7OpG7Em_kwIpucrg43Br64nGbeMM01yja4w")
+	if err != nil {
+		println(err.Error())
+	}
+	println(string(ret))
```

Comment:
Not sure that this would actually recover the data.

I tried 
```
data, _ := utils.Base64Decode(string(ret))
log.Println(string(data)) // {"keys":"123"}

```

Reply:
Hmmm yap, I think so. I have just made couple of transactions with raw data:

`	ret, err := arweave.Read("T54mWFQD2Y-0pB2wgxXJBCfXfxZisEL73G8Vmhtga9A")
	println(string(ret)) // {"keys":"123"}
`

`	ret, err := arweave.Read("Qaafc4sF-J7jqW7C8_smv0yPgN3WO2mzjZumMIAkbFg")
	println(string(ret)) // 123
`

Probably the one you queried was actually base64 encoded before uploading.